### ISSUE TITLE
Wintoolkit

### ISSUE THREAD

**Issue by acoul** [abbodi1406/dotNetFx4xW7A#7]
Greetings,

on the welcoming Readme it is mentioned:

"Mainly to be used with  [Wintoolkit](https://www.majorgeeks.com/files/details/win_toolkit.html)"

Installing Wintoolkit, on the first run, it suggests to download a new version of DISM which redirects you to a non-existing web page (win-unattended.de) which looks like a phishing area

**Comment by abbodi1406** [comment]
That link now point to the modified version of Wintoolkit, the last original version by original author is available here:
https://github.com/Legolash2o/WinToolkit_v1

[WTK_v1_1.7.0.0.7z](https://github.com/Legolash2o/WinToolkit_v1/raw/master/WTK_v1_1.7.0.0.7z) - require .NET 3.5
[WTK_v1_1.7.0.1.7z](https://github.com/Legolash2o/WinToolkit_v1/raw/master/WTK_v1_1.7.0.1.7z) - require .NET 4.8

i will update readme accordingly

**Comment by acoul** [comment]
I got the [WTK_v1_1.7.0.1.7z](https://github.com/Legolash2o/WinToolkit_v1/raw/master/WTK_v1_1.7.0.1.7z) with success. 

when Win Toolkit starts, it suggest to download a newer version of DISM and redirects you to [this page](https://www.wincert.net/leli55PK/DISM/) where the download links are broken.

**Comment by abbodi1406** [comment]
Ignore
